package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "armstrong.hatband",
-    "version": "1.4.0",
+    "version": "1.4.0tt1",
```

Comment:
For now I'd still lean towards keeping with this pattern for consistency's sake (and because we might be gutting armstrong apps before we upgrade pip anyway), but this is good to know and something to discuss and agree on a standard for when we upgrade.